README.md

```diff
@@ -17,11 +17,14 @@ gcloud compute disks create --size 30GB <disk name>
 Before creating a new cluster in GKE, a PosgreSQL database and role have to be created for it.
 This operation requires privileges for creating and deleting arbitrary databases in Cloud SQL.
 For security reasons the role used to access the database from within the cluster should not have such wide privileges and this step needs to be performed outside of cluster deployment process.
+This step require knowledge of cloud and cluster passwords to begin decrypting process.
 
 ``` bash
 cd concent-deployment/cloud/
 ansible-playbook create-databases.yml                              \
     --extra-vars cluster=$cluster                                  \
+    --vault-id @prompt                                             \
+    --ask-vault-pass                                               \
```

Comment:
From the docs I see that both `--vault-id @prompt` and `--ask-vault-pass` do the same thing. Let's settle on one and use it consistently. `--vault-id` seems to be newer and is now recommended so let's use it in all the commands instead of `--ask-vault-pass`.

Reply:
This playbook use two different password for cloud and cluster secrets. That situation in ansible not work with one flag.